src/store/CategoricalDisplayStore.js

```diff
@@ -130,7 +131,8 @@ export default class CategoricalDisplayStore {
         if (this.pauseScrollSpy || !this.scrollPanel) { return; }
         if (this.filteredItems.length === 0) { return; }
 
-        const categoryTitleHeight = 40;
+        // hits: 40px for title hight + 4px for content bottom border
+        const categoryTitleHeight = 44;
```

Comment:
this is not related to this card. during recent style modification a bottom border added to categorical display section bottom, which cause a jump in fix title.